jcl/src/java.base/share/classes/com/ibm/oti/util/Util.java

```diff
@@ -350,4 +365,21 @@ public static void printStackTraceElement(StackTraceElement e, Object elementSou
 	}
 }
 
+/**
+ * Check if a class loader is a built-in class loader.
+ *
+ * @param classLoader The classloader to check.
+ * @return true if the classloader provided is a built-in classloader,
+ * false otherwise.
+ */
+public static boolean isBuiltInClassLoader(ClassLoader classLoader) {
+	if ((null == classLoader) // Null or Bootstrap Class Loader
+			|| (ClassLoader.getSystemClassLoader().equals(classLoader))
+			|| (ClassLoader.getPlatformClassLoader().equals(classLoader))
```

Comment:
This won't compile on Java 8, `/*[IF Sidecar19-SE]*/` is needed around this function (but see the next comment).